repeated_word/repeated_word.py

```diff
@@ -0,0 +1,22 @@
+"""Find the first repeated word in a string."""
+
+
+def repeated_word(string):
+    """Find the first repeated word in a string."""
+    word = ''
+    words = [0] * len(string)
```

Comment:
Instead of building a list the length of the string, you can just `append()` each word found to an empty list.